validation/views.py

```diff
@@ -630,7 +630,6 @@ def prep_phrase_data(request, phrases):
 
 def save_issue(data, user):
     phrase_id = data["phrase_id"]
-    issues = data["issues"]
```

Comment:
Was _this_ what was causing a 500 error (due to a `KeyError`?)

Reply:
It sure was